### PR TITLE
Subscription and manage subscribers (CRUD&AUTH) #029

### PR DESCRIPTION
**Pull request by Salim-54**
#### What does this PR do?

- manages SUBSCRIPTIONS

#### Description of Task to be completed? 

As a user, I should be able to
- Subscribe to blogs 
As an admin, I should be able to
- Delete particular subscriber
- See all  subscribers

#### How should this be manually tested?

- Clone the repo

- checkout to branch ``ft-subscribe-CRUD#029``

- Run `npm install` to install modules

- Run `npm  run start` to start the development server

- In your API testing tool (postman recommended) call api `localhost:3000/api/v1/comments` to add a todo task

#### Any background context you want to provide?

- N/A

#### What are the relevant Pivotal Tracker/trello stories?

- https://trello.com/c/BRlBTO5f/44-subscriptions029

#### Screenshots (if appropriate)
- N/A

#### Questions:

- N/A

### CODE REVIEW

**Review by PrinceNiyonshuti — APPROVED** [review]
looks good with me

File: src/database/model/user.model.js [hunk 1]
```diff
@@ -28,18 +28,18 @@ const userSchema = new mongoose.Schema({
...
+  //   type: String,
+  //   required: [true, 'Please confirm your password'],
+  //   validate: {
+  //     // This only works on CREATE and SAVE!!!
+  //     validator: function(el) {
+  //       return el === this.password;
+  //     },
+  //     message: 'Passwords are not the same!'
+  //   }
+  // }
```
Comment (thepatrickniyo) [inline src/database/model/user.model.js:42]: Don't forget, commented source codes are not pushed for the best practice and code quality.

File: src/database/model/user.model.js [hunk 2]
```diff
@@ -50,8 +50,8 @@ userSchema.pre('save', async function(next) {
   this.password = await bcrypt.hash(this.password, 12);
 
   //delete pw
-  this.passwordConfirm = undefined;
-  next();
+  // this.passwordConfirm = undefined;
+  // next();
```
Comment (thepatrickniyo) [inline src/database/model/user.model.js:54]: Take care of comments here as well

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Fix all about the comments left in the source codes. 

One more thing is the space you leave between the two lines of code. Next time you can choose to leave a space where necessary only.

**Review by Xldivin — APPROVED** [review]

**Pull request merged**